docs/extend/modular-resources/examples/odrive.md

```diff
@@ -0,0 +1,281 @@
+
+In **Model**, enter either `viam:motor:odrive-serial` or `viam:motor:odrive-canbus`, depending on your preferred connection method.
+A warning tooltip will appear explaining this model is not built-in to the RDK, so it is not recognized by Viam and won't have validation features.
+
+![The ODrive motor component in the serial model being added as a component with name "your-odrive-serial" in the Viam app config builder](../../img/add-odrive/add-odrive-resource-ui.png)
+
+Click **Create component**.
+Edit and fill in the attributes as applicable to your model of ODrive.
+
+![The ODrive motor component in the serial model with example configuration attributes pasted in the Viam app config builder](../../img/add-odrive/add-odrive-attributes-ui.png)
+
+tab -->
+
+Go to your robot's page on the [Viam app](https://app.viam.com/).
+Navigate to the **Config** tab on your robot's page, and click on the **Components** subtab.
+Select **Raw JSON** mode.
+
+Copy and paste the following JSON depending on your preferred communication type:
+
+{{< tabs name="Add an ODrive motor">}}
+{{% tab name="odrive-serial" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+//  < INSERT YOUR ODRIVE MODULE CONFIG >
+  "components": [
+    {
+      "model": "viam:motor:odrive-serial",
+      "namespace": "rdk",
+      "attributes": {
+      },
+      "depends_on": [],
+      "type": "motor",
+      "name": "<your-odrive-motor>"
+    }
+  ]
+}
+```
+
+{{% /tab %}}
+{{% tab name="odrive-canbus" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+//  < INSERT YOUR ODRIVE MODULE CONFIG >
+{
+  "components": [
+    {
+      "model": "viam:motor:odrive-canbus",
+      "namespace": "rdk",
+      "attributes": {
+        "canbus_node_id": <int>,
+      },
+      "depends_on": [],
+      "type": "motor",
+      "name": "<your-odrive-motor>"
+    }
+  ]
+}
+```
+
+{{% /tab %}}
+{{% tab name="JSON Example" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "modules": [
+    {
+      "name": "odrive",
+      "executable_path": "/path/to/run.sh"
+    }
+  ],
+  "components": [
+    {
+      "model": "viam:motor:odrive-canbus",
+      "namespace": "rdk",
+      "attributes": {
+        "canbus_node_id": 0,
+        "odrive_config_file": "/path/to/first/config.json",
+        "serial_number": "NUM0001"
+      },
+      "depends_on": [],
+      "type": "motor",
+      "name": "odrive-motor"
+    },
+    {
+      "model": "viam:motor:odrive-canbus",
+      "namespace": "rdk",
+      "attributes": {
+        "canbus_node_id": 2,
+        "odrive_config_file": "/path/to/second/config.json",
+        "serial_number": "NUM0002"
+      },
+      "depends_on": [],
+      "type": "motor",
+      "name": "odrive-motor-2"
+    }
+  ]
+}
+```
+
+Edit and fill in the attributes as applicable to your model of ODrive.
+
+{{% /tab %}}
+{{< /tabs >}}
+
+The following attributes are available for the motor resources available in the Viam ODrive module:
```

Comment:
So, this module has two models with two different config files in the larger ODrive module, I think we should make that clear.

Reply:
@randhid That's a good point, didn't clearly distinguish-- trying to edit intro at least to clear that up

Reply:
let me know when you want me to look at the intro again.

Reply:
@randhid It's good to re-review now!